packages/quantic/force-app/examples/main/lwc/exampleQuanticCaseClassification/exampleQuanticCaseClassification.js

```diff
@@ -0,0 +1,58 @@
+import {LightningElement, track, api} from 'lwc';
+
+export default class ExampleQuanticCaseClassification extends LightningElement {
+  @api engineId = 'case-assist-engine';
+  @api caseAssistId = 'a4fb453a-b1f1-4054-9067-bef117586baa';
```

Comment:
What do we think of adding perhaps a utils function to `getDefaultCaseAssistId()` instead of hardcoding a string value directly here?

Reply:
On second thought it might make sense to keep it there and contained.

Reply:
Is there an API call that I can make to retrieve the case assist config ID ?

Reply:
[Here ](https://platformdev.cloud.coveo.com/docs/?urls.primaryName=Customer%20Service#/Configurations/getAllCaseAssistConfigurations)are all the case assist API calls. Yes there is one that lists all configs in an org. Keep in mind that the token needs certain privileges to successfully get this info. (It might already)

Reply:
Great! Thank you💪